photon-client/src/integTest/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriverIntegTest.scala

```diff
@@ -696,7 +696,7 @@ class GameTrainingDriverIntegTest extends SparkTestUtils with GameTestUtils with
 
 object GameTrainingDriverIntegTest {
 
-  private val TOLERANCE = 1E-6
+  private val TOLERANCE = 2E-5
```

Comment:
I would like to take a look at this myself before approving it.

Reply:
Certainly, the difference remains the same after persisting the RDDs. So I think this probably is due to the change of order from set to rdd.

Reply:
I just checked, this is fine. Actually, an upcoming change in one of my PRs had to reduce the tolerance even more.